src/components/Match/CastTable.jsx

```diff
@@ -55,16 +55,17 @@ const getCastArray = (pm) => {
   return resultArray;
 };
 
+// Deprecated & unused
```

Comment:
If it's not used, I'd just get rid of it. We can always pull it back out of the history.

Reply:
deleted it